features/upgrade/api_auth/upgrade.feature

```diff
@@ -122,3 +122,54 @@ Feature: apiserver and auth related upgrade check
       | <%= cb.oas_operator_image %> |
       | <%= cb.kas_image %>          |
       | <%= cb.oas_image %>          |
+
+  # @author kewang@redhat.com
+  @upgrade-prepare
+  @admin
+  Scenario: Default RBAC role, rolebinding, clusterrole and clusterrolebinding without any missing after upgraded - prepare
+    When I run the :get admin command with:
+      | resource | clusterroles.rbac |
+      | o        | yaml              |
+    Then the output should contain:
+      | autoupdate: "true" |
+    And the output should not contain:
+      | autoupdate: "false" |
+    # Make some changes on clusterrole resources before upgrade
+    Given as admin I successfully patch resource "clusterrole.rbac/system:build-strategy-docker" with:
+      | {"rules": [{"apiGroups": ["","build.openshift.io"],"resources": ["builds/docker","builds/optimizeddocker"],"verbs": [ "get" ]}] } |
```

Comment:
You should create a new clusterrole for this kind of test, or else other scenarios using this clusterrole will fail unexpectedly.

Reply:
@liangxia , as the case title, it exactly tests what will happen if the **default** RBAC things are changed. scheng will submit another PR for **default** SCC change, for which there was no case covered and thus caused **serious** customer problem (see Jira OCPQE-209 concerned by xiaoli)

Reply:
And this case @wangke19 auto'ing does not fail other scenarios, i.e. no harm.

Reply:
Adding @wzheng1 to confirm changing cluster-role "system:build-strategy-docker" does not break anything in devexp team.

Reply:
I just checked, it seems no influence to docker build.

Reply:
Ack